code-challenges.test.js

```diff
@@ -50,3 +162,34 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+
+// input: array
+// output: boolean value of true or false
+// create a function fullHouse that takes in an array of number.
+// declare a variable cardCount that has an empty object to store the number and the instances of the occurence.
+// use the .forEach method to iterate through each number. It will check if the number is in the object card count by accesing the value using square brackets if it is not in the object the it returns undefined by using the || it will assing the value of zero to that number then it adds 1 for each instance of occurence.
+// it will assing that value to the key inside of the object cardCount.
+// declare a variable cardInstance since our cardCount variable is an object with key pair values we can filter by using Object.values to look at each occurence.
+// using a method of .includes to check cardInstance has a value of 2 and then use a logical operator to evaluate if cardInstance also has a 3 in it. If both condtions are met then this will mean it is a full house three of a kind and a pair and return true. If it doesn not meet both conditions then it will return false.
+
+//  I found the way to store the count (object) of the numbers and  the logic for my forEach method form stack overflow post and used chat gpt to help explain it more detail in how the number was being counted and stored.
+// https://stackoverflow.com/questions/59444492/count-pairs-in-an-array-in-javascript
+
+const fullHouse = (array) => {
+  const cardCount = {}
+  array.forEach((number) => (cardCount[number] = (cardCount[number] || 0) + 1))
+  const cardInstance = Object.values(cardCount)
+  return cardInstance.includes(2) && cardInstance.includes(3)
+}
```

Comment:
Beautiful! Great research and great pseudocode explaining what you learned from your resources.